content/kit/wiring.md

```diff
@@ -1,3 +1,14 @@
 ---
 title: Wiring
 ---
+
+Ensuring your robot is wired correctly is incredibly important. If wired incorrectly, your kit may not work correctly, or may be damaged.
+
+## Wire Colouring
+Ground should be black, and nothing else should be black. This is in the interest of both safety and organisation. Being able to see where all your ground cables go allows you to easily track cable runs and diagnose electronic issues. It also greatly reduces the chance of running an actual current down the ground, which will seriously damage the kit!
```

Comment:
Changing the last sentence to something like

> It also greatly reduces the chance of misconnecting any power cables to ground, which would short-circuit the battery and seriously damage the kit!

would be more accurate.

(Note that if your ground connection is not carrying an "actual current", it is functionally equivalent to an open circuit and therefore might as well not exist :) )